examples/flutter_gallery/android/AndroidManifest.xml

```diff
@@ -3,7 +3,7 @@
      Use of this source code is governed by a BSD-style license that can be
      found in the LICENSE file.
  -->
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="io.flutter.MaterialGallery" android:versionCode="1" android:versionName="0.0.1">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="io.flutter.FlutterGallery" android:versionCode="1" android:versionName="0.0.1">
```

Comment:
I believe java is case sensitive here, but maybe we should take this opportunity to make the ios and android ones match?